js/spreadsheet.js

```diff
@@ -242,6 +242,23 @@ var spreadsheet = function(options) {
     },
     afterSelectionEnd: function(r, c, r2, c2) {
       s = [r, c, r2, c2];
+    },
+    beforeKeyDown: function (event) {
+      event = event || window.event;
+
+      if ((event.ctrlKey || event.metaKey) && event.keyCode === 65 ) {
+        event.stopImmediatePropagation();
+        
+        var cols = getColumns().filter(function(column) {
+          return column;
+        }).length
+        var colEnd = cols ? cols - 1: cols;
```

Comment:
This would an easier way to understand this.

```suggestion
        var colEnd = Math.max(cols - 1, 0);
```